deid/dicom/header.py

```diff
@@ -46,7 +46,7 @@
 import dateutil.parser
 import tempfile
 
-from .utils import get_func, save_dicom
+from .utils import get_func, save_dicom, setup_deid
 from .actions import perform_action
 from pydicom.dataset import Dataset
```

Comment:
This function should live with the others to load the deid, not with general utils.